manifests/params.pp

```diff
@@ -195,11 +195,14 @@
 
   # Puppet service name
   $service_name = 'puppet'
-  $agent_restart_command = $::osfamily ? {
-    'Debian' => '/usr/sbin/service puppet reload',
-    'Redhat' => '/usr/sbin/service puppet reload',
-    default  => undef,
+  # Command to restart the agent
+  if $::osfamily == 'Debian' { $agent_restart_command = "/usr/sbin/service $service_name reload" }
```

Comment:
Note that tests are failing as `$service_name` should use brackets, e.g. `${service_name}`